App/Species/KineticSpecies.lua

```diff
@@ -685,7 +705,7 @@ end
 function KineticSpecies:checkPositivity(tCurr, idx)
   local status = true
   if self.positivity then
-     status = self.posChecker:advance(tCurr, {self:rkStepperFields()[idx]}, {})
+     --status = self.posChecker:advance(tCurr, {self:rkStepperFields()[idx]}, {})
```

Comment:
Is this meant to stay commented out? if so, why not remove it?

Reply:
Thanks for catching that. I meant to delete.